frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -108,11 +206,29 @@ pods:
         {{/TASKCFG_ALL_SECURITY_TRANSPORT_ENCRYPTION_ENABLED}}
       repair:
         goal: ONCE
-        cmd: export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/) && ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/nodetool repair -pr -p ${JMX_PORT} -- $CASSANDRA_KEYSPACE
+        cmd: |
+              export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/)
+              {{#TASKCFG_ALL_SECURE_JMX_ENABLED}}
+              export HOME=$MESOS_SANDBOX
+              user=$(sed -nr '0,/re/s/.*(^)(.+) readwrite.*/\2/p' $MESOS_SANDBOX/jmx/access_file)
+              ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/nodetool --ssl -u $user -pwf $MESOS_SANDBOX/jmx_properties/key_file repair -pr -p ${JMX_PORT} -- $CASSANDRA_KEYSPACE
+              {{/TASKCFG_ALL_SECURE_JMX_ENABLED}}
+              {{^TASKCFG_ALL_SECURE_JMX_ENABLED}}
+              ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/nodetool repair -pr -p ${JMX_PORT} -- $CASSANDRA_KEYSPACE
+              {{/TASKCFG_ALL_SECURE_JMX_ENABLED}}
         resource-set: sidecar-resources
       cleanup:
         goal: ONCE
-        cmd: export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/) && ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/nodetool cleanup -p ${JMX_PORT} -- $CASSANDRA_KEYSPACE
+        cmd: |
```

Comment:
do we have any tests for these commands? these are the only places where we are enabling secure JMX options and if we break them in the future would be hard to detect.  
Can we add a test with secure JMX options enabled and run a cleanup for example?

Reply:
Yes @zmalik , i had discussion with Sam . i am currently working on to add test where jmx code is added like in task repair/cleanup/snapshot/cleanup-snapshot.